toolchains/regenerate.sh

```diff
@@ -33,10 +33,15 @@ fi
 if [[ "true" == "${COMMIT_TOOLCHAINS}" ]]; then
   COMMIT_MSG="Regenerate toolchains from $(git rev-parse HEAD)
 
+  [skip ci]
```

Comment:
What is this?

Reply:
because CI will push back to same repo, `[skip ci]` in commit message to skip CI run again from the automated commit.

Reply:
Can you add a comment above?

Reply:
done